src/context/appContext.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export const ColorModeContext = React.createContext({
```

Comment:
You can make this code slightly more clean by importing createContext directly:
```
import React, { createContext } from "react";
```
Then you don't need to write React.createContext you can directly use createContext({})